apps/pigment-css-next-app/package.json

```diff
@@ -9,7 +9,7 @@
     "clean": "rimraf .next"
   },
   "dependencies": {
-    "@pigment-css/react": "^0.0.12",
+    "@pigment-css/react": "^0.0.13",
```

Comment:
same

Reply:
The latest for `@pigment/react` is `0.0.13`. Only `@pigment-css/nextjs-plugin` that's get updated in `0.0.14`